lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -97,12 +96,14 @@ def capture(money, intent_id, options = {})
           post[:transfer_data][:amount] = options[:transfer_amount]
         end
         post[:application_fee_amount] = options[:application_fee] if options[:application_fee]
+        options = format_idempotency_key(options, 'capture')
         commit(:post, "payment_intents/#{intent_id}/capture", post, options)
       end
 
       def void(intent_id, options = {})
         post = {}
         post[:cancellation_reason] = options[:cancellation_reason] if ALLOWED_CANCELLATION_REASONS.include?(options[:cancellation_reason])
+        options = format_idempotency_key(options, 'cancel')
```

Comment:
🤔I was planning to handle this within Core failure cleanup since `void` is usually an independent action. Should we limit the scope to actions that are compound in AM or Core?

Reply:
ah yes, that makes sense - i will update 👍